com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -1030,7 +1031,7 @@ internal NetworkObject CreateLocalNetworkObject(NetworkObject.SceneObject sceneO
         /// Distributed Authority:
         /// DAHost client and standard DA clients invoke this method.
         /// </summary>
-        internal void SpawnNetworkObjectLocally(NetworkObject networkObject, ulong networkId, bool sceneObject, bool playerObject, ulong ownerClientId, bool destroyWithScene)
+        internal void AuthorityLocalSpawn(NetworkObject networkObject, ulong networkId, bool sceneObject, bool playerObject, ulong ownerClientId, bool destroyWithScene)
```

Comment:
Just an idea, we can add a `[NotNull]` compiler check on the `NetworkObject` parameter and save on the null check

```suggestion
        internal void AuthorityLocalSpawn([NotNull] NetworkObject networkObject, ulong networkId, bool sceneObject, bool playerObject, ulong ownerClientId, bool destroyWithScene)
```